app/api/admin_sales/fees.py

```diff
@@ -22,13 +22,14 @@ class Meta:
     name = fields.String()
     payment_currency = fields.String()
     fee_percentage = fields.Float(attribute='fee')
+    maximum_fee = fields.Float(attribute='maximum_fee')
```

Comment:
@iamareebjamal this variable is included to provide info on the frontend regarding max fees.
![Screenshot_2019-03-23 Fees Sales Administration Open Event(2)](https://user-images.githubusercontent.com/25428397/54869196-44341d80-4dbb-11e9-96a4-5f0a11c7719a.png)